### ISSUE TITLE
Livedo racemosa

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6857]
**Preferred term label:**
Livedo racemosa

**Synonyms**
Vasculitis racemosa

**Definition (free text, please give PubMed ID)**
Livedo racemosa is a cutaneous finding characterized by a persistent, erythematous, or violaceous discoloration of the skin, in a broken, branched, discontinuous, and irregular pattern.
DOI: https://doi.org/10.1177/1534734619896938

**Parent term (use hpo.jax.org/app)**
Erythema

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM: # 182410

**Comment by pnrobinson** [comment]
This term already exists -- https://hpo.jax.org/app/browse/term/HP:0033260
I will add the link to OMIM:182410, thanks!
